applications/FluidDynamicsBiomedicalApplication/python_scripts/apply_parabolic_inlet_process.py

```diff
@@ -0,0 +1,117 @@
+# Importing the Kratos Library
+import KratosMultiphysics
+
+# Import applications
+import KratosMultiphysics.FluidDynamicsBiomedicalApplication as KratosBio
+
+def Factory(settings, model):
+    if(type(settings) != KratosMultiphysics.Parameters):
```

Comment:
The new way would be to use `isinstance`

Reply:
Done. Copy and pasting leftover.